staff/alex-park/precourse/pasapalabra/index.js

```diff
@@ -0,0 +1,147 @@
+    { letter: "t", answer: ["terminator","trece","techo"], status: 0, question: ["CON LA T. Película del director James Cameron que consolidó a Arnold Schwarzenegger como actor en 1984","CON LA T. Número cardinal equivalente a 10 + 3","CON LA T. Parte de una habitación que está arriba"]},
+    { letter: "u", answer: ["unamuno","uderzo","urgente"], status: 0, question: ["CON LA U. Escritor y filósofo español de la generación del 98 autor del libro 'Niebla' en 1914","CON LA U. Apellido del dibujante y guionista francés autor de la serie Asterix","CON LA U. Que no puede esperar"]},
+    { letter: "v", answer: ["vikingos","verde","veloz"], status: 0, question: ["CON LA V. Nombre dado a los miembros de los pueblos nórdicos originarios de Escandinavia, famosos por sus incursiones y pillajes en Europa","CON LA V. Se aplica el color perfectamente al de la hierba fresca o la esmeralda","CON LA V. Que es muy rápido"]},
+    { letter: "w", answer: ["sandwich","wakanda","wonder"], status: 0, question: ["CONTIENE LA W. Emparedado hecho con dos rebanadas de pan entre las cuales se coloca jamón y queso","CON LA W. Tierra natal de Black Panther","CON LA W. Apellido del cantante de nombre artístico Stevie viral por su clásico Superstition"]},
+    { letter: "x", answer: ["botox","exérgesis","execrable"], status: 0, question: ["CONTIENE LA X. Toxina bacteriana utilizada en cirujía estética","CONTIENE LA X. Explicación o interpretación particularmente de los libros de la Biblia","CONTIENE LA X. Deleznable, repudiable o digno de condena"]},
+    { letter: "y", answer: ["peyote","moncayo","yegua"], status: 0, question: ["CONTIENE LA Y. Pequeño cáctus conocido por sus alcaloides psicoactivos utilizado de forma ritual y medicinal por indígenas americanos","CONTIENE LA Y. Nombre de la montaña más alta del sistema ibérico","CON LA Y. Hembra del caballo"]},
+    { letter: "z", answer: ["zen","zoodiacal","zarpar"], status: 0, question: ["CON LA Z. Escuela de budismo que busca la experiencia de la sabiduría más allá del discurso racional","CON LA Z. Perteneciente o relativo al zoodíaco","CON LA Z. Empezar a navegar"]},
+];
+
+var score = 0;
+var statusCounter = 0;
+var nameTracker;
+var ranking = [
+    {name: "Obama", score: 20},
+    {name: "Ralph", score: 9},
+    {name: "Donald Trump", score: 5}
+];
+var temp;
+var endTracker = false;
+
+function pasapalabra () {
+    var name = prompt("¡Bienvenido a Pasapalabra! Por favor, indícanos tu nombre:");
+    nameTracker = {name: name, score: 0};
+    var starter = confirm("Muy buenas, " + name + ", a continuación se te mostrará el actual Top 3 de jugadores:\n\n1º " + ranking[0].name + ": " + 
+    ranking[0].score + " puntos\n2º " + ranking[1].name + ": " + ranking[1].score + " puntos\n3º " + ranking[2].name + ": " + ranking[2].score +
+    " puntos\n\nCuando estés preparad@ dale a 'Aceptar' y ¡que empiece el juego!");
+
+    if (starter === false) {
+        alert("Has tirado la toalla sin ni siquiera intentarlo :^(");
```

Comment:
Hey Alex, I see that you comunicate every message to the user with alert. Why don't you try to display that message somewhere on the HTML? For example on the "maintitle" class on the HTML. Tell me what you think!

Reply:
Hey Lisandro, even though you're pointing out the first pasapalabra project, you're absolutely right! I'll try to avoid using alerts from now on :)